src/Tribe/Tickets.php

```diff
@@ -2494,6 +2503,12 @@ public function send_tickets_email_for_attendee( $to, $tickets, $args = [] ) {
 				// Mark attendee ticket email as being sent for each attendee ticket.
 				foreach ( $tickets as $attendee ) {
 					$this->update_ticket_sent_counter( $attendee['attendee_id'], $this->attendee_ticket_sent );
+
+					$this->update_attendee_activity_log( $attendee['attendee_id'], [
+						'type'  => 'email',
+						'name'  => $attendee['holder_name'],
+						'email' => $attendee['holder_email'],
+					] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: bf6bd481039add0061fc6a64b31a5cea</sub>